container_files/assets/www/doc/builtin/functions/SqlQueryFunction.md

```diff
@@ -25,15 +28,59 @@ put them inside '"' characters.  For example,
 - `$x` refers to the input value named `x`.
 - `$"x and y"` refers to the input value named `x and y`.
 
+The input values are available in the entire expression, including
+as part of table expressions or join expressions.
+
+## Output format
+
+There are two possibilities for the output:
+
+1.  If the `output` field is set to `FIRST_ROW` (the default), then
+    it will outputs the `SELECT` expression applied to the first matching
+    row (ie, with an `OFFSET` of 0 and a `LIMIT` of 1).  If multiple
+    rows are matched, all but the first will be ignored.
+2.  If the `output` field is set to `NAMED_COLUMNS`, then the query
+    must return a two-column dataset with a `column` and a `value`
+    each column.  Each row  in the output will generate a single
+    column in the output row, with the column name equal to value of the
+    `column` column, and the column value equal to the value of the
+    `value` column.  In this case the `OFFSET` and `LIMIT` will be
+    respected.  This allows for more sophisticated rows to be returned
+    from queries, and is especially useful in conjunction with joins.
```

Comment:
I'm not sure I understand the use-case here... Can we see a real-world application?